services/ecosystems/src/routes/ecosystems.js

```diff
@@ -0,0 +1,202 @@
+const { Router } = require('restify-router');
+
+module.exports = (errorHandler, EcosystemsService) => {
+  const router = new Router();
+
+  /**
+   * @apiGroup s_coverage
+   * @api {get} /coverage Coverage
```

Comment:
Update the endpoint definition and example

Reply:
Done!